site/snippets/layouts/banner.php

```diff
@@ -1,9 +1,5 @@
-<?php if (empty($banner) === false && empty($banner['text']) === false): ?>
+<?php if (empty($banner) === false && $banner->text()): ?>
 <aside class="banner text-sm">
-  <?php if (isset($banner['url'])): ?>
-  <a href="<?= $banner['url'] ?>"><?= $banner['text'] ?></a>
-  <?php else: ?>
-  <span><?= $banner['text'] ?></span>
-  <?php endif ?>
+  <a href="<?= url('buy') ?>"><?= $banner->text() ?></a>
```

Comment:
I was thinking whether shouldn't leave it more flexible, e.g. to also be able to have a banner for an upcoming live stream. But I guess you went this way due to the current styling (pointing to Buy)?

Reply:
Yep. I thought about adding a new option `isSale` to control the special placement on the buy page, bit given that the banner always points to buy, I think it doesn't make much sense to use it for other purposes besides sales.

Reply:
@bastianallgeier What do you think about this? It could make sense to use the banner for different purposes than sales as Nico mentioned (live streams etc.). In this case we'd need a styling variant where the banner doesn't appear next to the Buy link though. This styling variant could be controlled automatically in the snippet depending on the target URL.